### ISSUE TITLE
How do I change the path in prepare_labels.py

### ISSUE THREAD

**Issue by BigBrowww** [sovit-123/Traffic-Light-Detection-Using-YOLOv3#5]
Hello and good evening.
When I changed the path in prepare_lables.py after installing the dataset, I found that there is no such path like [ input/labels/{image_name.split('.')  [0]}.txt ] or [ input/images/{image_name} ]  in the folder of the dataset,.
waiting for your reply, thank you!

**Comment by sovit-123** [comment]
Hello. Please note that the dataset is one folder back in the `input` directory. Therefore, it is `../input/rest_of_the_path`. Can you please make sure it is the same case in your code as well?

**Comment by BigBrowww** [comment]
Hi
Here's the thing, I didn't run the code on Kaggle, so I extract the dataset to my hard drive, and the path is E:\Archive, and in the Archive folder is Anovation, DayTrain, etc., so I'm having some trouble figuring out how to change the path in prepare_labels.py.

**Comment by BigBrowww** [comment]
Yup, this error makes me confuse. I have changed annotation_root and image_root.But I dont know how to get the *.txt
![``UCC Z8(3~{H0CQ69BKT`P](https://user-images.githubusercontent.com/49865072/168149248-14442c52-ed6d-4a71-948d-be0747658623.png)

**Comment by sovit-123** [comment]
Can you try giving relative path like the code expects? The code in this repo does not expect Kaggle environment but local system. So, having the data one folder back in the input directory that is `../input` should work without issues.

**Comment by HASSINASADDEDINESegula** [comment]
hello, i have a same probleme :  FileNotFoundError: [Errno 2] No such file or directory: 'C:/Users/SADDEDINE_H/Desktop/Trafic Lights/Data/label/dayClip1--00000.txt'
I downloaded all the LISA data I deposited it in the desktop but I can't find the ".txt"

**Comment by Jiachengc320** [comment]
> Yup, this error makes me confuse. I have changed annotation_root and image_root.But I dont know how to get the *.txt ![``UCC Z8(3~{H0CQ69BKT`P](https://user-images.githubusercontent.com/49865072/168149248-14442c52-ed6d-4a71-948d-be0747658623.png)

You should create image and labels folders in those directories so you can run them
